pkg/ibmcsidriver/server_test.go

```diff
@@ -20,13 +20,30 @@
 package ibmcsidriver
 
 import (
+	"errors"
 	"flag"
+	"fmt"
+	"os"
 	"testing"
 
 	cloudProvider "github.com/IBM/ibmcloud-volume-file-vpc/pkg/ibmcloudprovider"
 	"github.com/stretchr/testify/assert"
 )
 
+// mockFileOps is a mock implementation of FileOps for testing setupSidecar
+type mockFileOps struct {
+	chownErr error
+	chmodErr error
+}
+
+func (m *mockFileOps) Chown(name string, uid, gid int) error {
```

Comment:
generate mock tests instead of writing own

Reply:
Done